src/main/java/de/numcodex/feasibility_gui_backend/terminology/api/AttributeDefinition.java

```diff
@@ -26,11 +31,13 @@ public class AttributeDefinition {
     @JsonProperty("optional")
     private Boolean optional;
     @JsonProperty("allowedUnits")
-    private List<Unit> allowedUnits = new ArrayList<>();
+    private List<TermCode> allowedUnits = new ArrayList<>();
```

Comment:
is this really supposed to be TermCode and not Unit - please check with @geloro94

Reply:
I would move that question to a new issue and merge this one here.